tensorflow/python/eager/def_function_test.py

```diff
@@ -339,7 +339,7 @@ def sum(self):
         # Use a loop & conditional to test that autograph works correctly.
         result = 0
         for i in range(array_ops.size(self.values)):
-          if self.mask[i]:
+          if self.mask[i] and 0 <= self.values[i] <= 3:
```

Comment:
This one is going to cover the integration at high level. Probably it could be enough if we don't care about something specific in the lowlevel.

Reply:
I don't this is the right test to modify, it tests something else. We can use the internal integration tests once this PR is submitted.

Reply:
Ah, I missed the comment. SGTM.

Reply:
Hi I've removed this highlevel integration test just cause I don't see any other simple test on conditional, logic etc.. there.